src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtGeneralInfoEndpointTest.java

```diff
@@ -0,0 +1,173 @@
+package uk.gov.hmcts.dts.fact.admin;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.model.Court;
+import uk.gov.hmcts.dts.fact.model.admin.CourtGeneralInfo;
+import uk.gov.hmcts.dts.fact.util.AdminFunctionalTestBase;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.HttpStatus.*;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.BEARER;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.COURTS_ENDPOINT;
+
+@ExtendWith(SpringExtension.class)
+public class AdminCourtGeneralInfoEndpointTest extends AdminFunctionalTestBase {
+    private static final String ADMIN_COURT_GENERAL_INFO_PATH = "/generalInfo";
+    private static final String BIRMINGHAM_CIVIL_AND_FAMILY_JUSTICE_CENTRE_SLUG = "birmingham-civil-and-family-justice-centre";
+    private static final CourtGeneralInfo EXPECTED_ADMIN_COURT_INFO = new CourtGeneralInfo(
+        true,
+        false,
+        "Admin Info",
+        "Welsh Admin Info",
+        "Admin Alert",
+        "Welsh Admin Alert"
+    );
+    private static final CourtGeneralInfo EXPECTED_SUPER_ADMIN_COURT_INFO = new CourtGeneralInfo(
+        true,
+        false,
+        "Super Admin Info",
+        "Super Welsh Admin Info",
+        "Super Admin Alert",
+        "Super Welsh Admin Alert"
+    );
+
+    private static String adminCourtInfoJson;
+
+    @BeforeAll
+    static void initialise() throws JsonProcessingException {
+        adminCourtInfoJson = new ObjectMapper().writeValueAsString(EXPECTED_ADMIN_COURT_INFO);
```

Comment:
Could this be initialised on line 42? In one go, as a final variable? Or is this because it throws the JsonProcessingException? If so, then no worries and resolve :)

Reply:
Yeah that's right. I was initially adding this as a static final variable but because of the exceptionIt I am doing it this way :)